src/_cffi_src/openssl/ssl.py

```diff
@@ -688,6 +689,12 @@
 
 static const long Cryptography_HAS_SSL_CTX_CLEAR_OPTIONS = 1;
 
+#ifdef OPENSSL_NO_TLSEXT
```

Comment:
We use e.g. SSL_CTX_set_tlsext_servername_callback, I'm not sure if that is to be called without the macro OPENSSL_NO_TLSEXT defined.